tests/e2e/network/service_annotations.go

```diff
@@ -36,12 +40,21 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+var (
+	scalesetRE = regexp.MustCompile(`.*/subscriptions/(?:.*)/resourceGroups/(.+)/providers/Microsoft.Compute/virtualMachineScaleSets/(.+)/virtualMachines(?:.*)`)
+	lbNameRE   = regexp.MustCompile(`^/subscriptions/(?:.*)/resourceGroups/(?:.*)/providers/Microsoft.Network/publicIPAddresses/(.+)`)
```

Comment:
should be `regexp.MustCompile(`^/subscriptions/(?:.*)/resourceGroups/(?:.*)/providers/Microsoft.Network/loadBalancers/(.+)/frontendIPConfigurations(?:.*)`)`

Reply:
okay I used this example for setting that
`"publicIPAddress": {
                "id": "/subscriptions/11a9a54c-e5aa-43b1-85f5-ce4294e0dbaa/resourceGroups/akscheck/providers/Microsoft.Network/publicIPAddresses/k8s-master-ip-k8si-26706573"
              }`

Reply:
lbName can only be got from frontendIPConfigurationID, not publicIP's ID